23-07-workshop/in-progress/query_region_plot_HR_plot_lc.ipynb

```diff
@@ -0,0 +1,1082 @@
+{
```

Comment:
<p>We should keep this info, since it's good to know how bokeh works.</p><p> </p><p>I wonder if we should also talk about the actual HR diagram; is this the shape we expect, interesting features, etc</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Assumption: faculty member would have already covered the HR diagram. Maybe just add "this is main sequence"</p><p> </p><p>Add prereq: some familiarity with HR diagram</p>